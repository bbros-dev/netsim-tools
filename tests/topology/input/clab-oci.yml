provider: clab
name: topology
defaults:
  device: frr

nodes:
- name: t1
- name: t2

links:
- t1-t2
