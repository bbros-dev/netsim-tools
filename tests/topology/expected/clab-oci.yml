name: topology

topology:
  nodes:
    t1:
      kind: linux
      image: frrouting/frr:v7.5.0
    t2:
      kind: linux
      image: frrouting/frr:v7.5.0

  links:
  - endpoints:
    - "t1:eth1"
    - "t2:eth1"
